pandas/tests/extension/test_datetime.py

```diff
@@ -88,6 +88,13 @@ def cmp(a, b):
     return cmp
 
 
+@pytest.fixture
+def isnumeric(dtype):
+    return getattr(dtype, "is_numeric", False) or issubclass(
+        np.dtype(dtype).type, np.number
+    )
```

Comment:
pytest fixtures aren't themselves tests, they are used to help in the creation of tests. In other words, this alone does not test anything. The test should likely go in `tests/frame/test_query_eval.py`.